cli/src/semgrep/app/scans.py

```diff
@@ -272,10 +272,18 @@ def report_findings(
         ignores = [
             match.to_app_finding_format(commit_date).to_json() for match in new_ignored
         ]
+        token = (
+            # GitHub (cloud)
+            os.getenv("GITHUB_TOKEN")
+            # GitLab.com (cloud)
+            or os.getenv("GITLAB_TOKEN")
+            # Bitbucket Cloud
+            or os.getenv("PIPELINES_JWT_TOKEN")
+        )
+
         findings_and_ignores = {
             # send a backup token in case the app is not available
-            "token": os.getenv("GITHUB_TOKEN"),
-            "gitlab_token": os.getenv("GITLAB_TOKEN"),
```

Comment:
I will leave the corresponding app code that still detects GitLab tokens like this for now until we make a decision later to stop supporting x versions back of Semgrep.